src/network/MissionControlProtocol.h

```diff
@@ -72,6 +73,7 @@ class MissionControlProtocol : public WebSocketProtocol { // TODO: add documenta
 	void handleHeartbeatTimedOut();
 	void startPowerRepeat();
 	void stopAndShutdownPowerRepeat();
+	void setArmIKEnabled(const bool enabled);
```

Comment:
There's no point in `const` pass-by-value arguments, so we should just make this `bool enabled`